crates/rspack_plugin_swc_js_minimizer/src/lib.rs

```diff
@@ -36,6 +36,7 @@ static JAVASCRIPT_ASSET_REGEXP: LazyLock<Regex> =
 
 #[derive(Debug, Hash)]
 pub struct PluginOptions {
+  pub minify: Option<bool>,
```

Comment:
Let's put this into minimizerOptions, since this is an option pass to swc minifier instead of consuming by the plugin